bfsdk/cmake/CMakeGlobal_Project.txt

```diff
@@ -16,147 +16,33 @@
 # License along with this library; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
 
-# ------------------------------------------------------------------------------
-# Includes
-# ------------------------------------------------------------------------------
-
-if(NOT DISABLE_CTEST)
-    include(CTest)
```

Comment:
CTest needs to be disabled for the super builds because they do not have unit tests.